### ISSUE TITLE
0.8.X docker images doesn't work as expected.

### ISSUE THREAD

**Issue by ajnavarro** [pilosa/pilosa#1060]
### Expected behavior
Following step by step the [Getting Started guide](https://www.pilosa.com/docs/latest/getting-started/), be able to execute `curl localhost:10101/status` and get a json response using the latest pilosa docker container.

### Actual behavior

``` bash
14:35 $ curl http://localhost:10101/status
curl: (56) Recv failure: Connection reset by peer
```

### Steps to reproduce the behavior
- Execute: `docker run -it --rm --name pilosa -p 10101:10101 pilosa/pilosa:latest`
- Then, when the container is running pilosa, execute: `curl http://localhost:10101/status`

Note that this only happens with 0.8.X versions. With previous versions like 0.7.X is working properly.

Creating the image from source code using `docker build .` is generating a correct docker image. `make docker` is working correctly too.

### Information about your environment (OS/architecture, CPU, RAM, cluster/solo, configuration, etc.)
`Linux XXX-XPS-15-9550 4.4.0-109-generic #132-Ubuntu SMP Tue Jan 9 19:52:39 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by yuce** [comment]
Thanks for reporting this. The `Dockerfile` was updated to fix this, but it wasn't pushed to Docker hub. Can you add `-e PILOSA_BIND=0.0.0.0:10101` to your docker commandline in the meantime? Like:
```
docker run -it --rm --name pilosa -p 10101:10101 -e PILOSA_BIND=0.0.0.0:10101 pilosa/pilosa:latest
```

**Comment by codysoyland** [comment]
Thanks @ajnavarro. We are working on pushing a release that automatically binds the Pilosa port, but @yuce's suggestion should work in the meantime.

This was previously reported as #977 and fixed in #980, but somehow I missed applying it to the release branch, so it was never actually pushed to Docker Hub. The v0.8.5 release should fix this, which I will push after we close out #1061.

**Comment by codysoyland** [comment]
@ajnavarro Release 0.8.5 is now live on Docker Hub and seems to work with the documented command. You can upgrade with `docker pull pilosa/pilosa`. Thanks for your report, and let me know if you continue to have any issues!
